### PR TITLE
Migrate pause/stop/resume/abort endpoints

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala [hunk 1]
```diff
@@ -38,56 +38,45 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
...
     case req @ POST -> Root / ObsIdVar(oid) / StepIdVar(step) / ClientIDVar(clientId) / "execute" /
         ResourceVar(resource) / ObserverVar(obs) =>
-      ssoClient.require(req) { user =>
-        oe.configSystem(oid, obs, user, step, resource, clientId) *>
-          NoContent()
-      }
+      ssoClient.require(req): user =>
+        oe.configSystem(oid, obs, user, step, resource, clientId) *> NoContent()
+
+    case req @ POST -> Root / ObsIdVar(obsId) / StepIdVar(stepId) / "startFrom" /
```
Comment (cquiroz) [inline modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:50]: It seems we have used `obsid/stepid/clientid/<verb>` format in other endpoints, should we make them match?

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala [hunk 2]
```diff
@@ -38,56 +38,45 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
...
+          stepId,
+          clientId,
+          runOverride.getOrElse(RunOverride.Default)
+        ) *> NoContent()
+
+    case req @ POST -> Root / ObsIdVar(obsId) / "pause" / ObserverVar(obs) =>
+      ssoClient.require(req): user =>
+        oe.requestPause(obsId, obs, user) *> NoContent()
+
+    case req @ POST -> Root / ObsIdVar(obsId) / "cancelpause" / ObserverVar(obs) =>
```
Comment (cquiroz) [inline modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:65]: `cancelPause`? to match `startFrom`?

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala [hunk 3]
```diff
@@ -100,101 +89,86 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
...
-    //     ) as user =>
-    //   se.pauseObserve(inputQueue, obsId, obs, user, graceful = true) *>
-    //     Ok(s"Pause observation gracefully requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsIdVar(obsId) / StepId(stepId) / "resumeObs" / ObserverVar(obs) as user =>
-    //   se.resumeObserve(inputQueue, obsId, obs, user) *>
-    //     Ok(s"Resume observation requested for $obsId on step $stepId")
-    //
+
+    case req @ POST -> Root / ObsIdVar(obsId) / StepIdVar(stepId) / "stop" /
```
Comment (cquiroz) [inline modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:93]: I think we should include `clientId` in all endpoints, even if it just for logging

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:50]: Sure, I'll rearrange

File: modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala (hunk 3 above)
Comment (rpiaggio) [reply modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala:93]: Good idea

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**